### PR TITLE
File Added : Uploading asignment5 and 6

### PR DESCRIPTION
**Pull request by arshad05786**

### CODE REVIEW

File: arshad_khan/html/asignment5/asignment5quest.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                        </li>
+                        <li>
+                            <a href="gallery.html" target="_blank">Photo Gallery</a>
+                        </li>
+                    </ul>
+                </nav>
+            </section>
+        </div>
+    </body>
+</html>
```
Comment (yogesh141293) [inline arshad_khan/html/asignment5/asignment5quest.html:41]: Add extra line

File: arshad_khan/html/asignment6/firstquest.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                </label>
+                <h3><i>Academic Details</i></h3>
+                <label for="skill"
+                    >Skill Set <input type="checkbox" name="skill" value="html" />HTML <input type="checkbox" name="skill" value="javascript" /> Java Script
+                    <input type="checkbox" name="skill" value="java" />Java <input type="checkbox" name="skill" value="oracle" />Oracle
+                </label>
+            </form>
+        </div>
+    </body>
+</html>
```
Comment (yogesh141293) [inline arshad_khan/html/asignment6/firstquest.html:39]: Add Extra Line

**Review by yogesh141293 — COMMENTED** [review]
Add extra lines in all Assigments

**Review by SamikshaWani — APPROVED** [review]

**Review by Oshin-250197 — APPROVED** [review]

**Review by roshnidurugwar — APPROVED** [review]

**Review by yogesh141293 — APPROVED** [review]

File: arshad_khan/html/asignment6/firstquest.html [hunk 3]
```diff
@@ -21,7 +21,8 @@ <h3><i>Personel Details</i></h3>
                 <label for="lname">Last Name <input type="text" placeholder="Last Name" /> </label><br /><br />
```
Comment (SamikshaWani) [inline arshad_khan/html/asignment6/firstquest.html]: placeholders second word first letter should be small

File: arshad_khan/html/asignment6/firstquest.html [hunk 4]
```diff
@@ -21,7 +21,8 @@ <h3><i>Personel Details</i></h3>
...
                 <label for="gender"
                     >Gender <input type="radio" name="gender" value="male" />Male <input type="radio" name="gender" value="female" />Female
```
Comment (SamikshaWani) [inline arshad_khan/html/asignment6/firstquest.html]: check indentation

File: arshad_khan/html/asignment6/firstquest.html [hunk 5]
```diff
@@ -21,7 +21,8 @@ <h3><i>Personel Details</i></h3>
...
-                    <input type="radio" name="gender" value="other" />Other </label>
+                    <input type="radio" name="gender" value="other" />Other
+                </label>
                 <br /><br />
                 <label for="age"
                     >Age <input type="radio" name="age" value="lessthan18" />Less than 18 <input type="radio" name="age" value="18-25" />18-25
```
Comment (SamikshaWani) [inline arshad_khan/html/asignment6/firstquest.html]: check indentation

File: arshad_khan/html/asignment5/asignment5quest.html [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+        <link rel="stylesheet" href="asign.css" />
+    </head>
+    <body>
+        <div class="divmain">
+            <header>
+                <h1>MOHD ARSHAD KHAN</h1>
+            </header>
+            <i>
+                <h1 class="heading1">About Me</h1>
+            </i>
```
Comment (Pawan1005) [inline arshad_khan/html/asignment5/asignment5quest.html]: use inside of h1 tag

File: arshad_khan/html/asignment5/asignment5quest.html [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+            <p class="p1">
+                My name is Mohd Arshad Khan. <br />I am from UP.<br />I have completed my post graduate from Maulana Azad National Urdu University Hyderabad.<br />
+                I have completed my under graduation from Integral University Lucknow.<br /><br /><br /><br />
+            </p>
+            <div class="div2">
+                <br /><br /><br /><br /><br />
+                <i>
+                    <h2>More About Me</h2>
+                </i>
```
Comment (Pawan1005) [inline arshad_khan/html/asignment5/asignment5quest.html]: use inside of h2 tag

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**